provision-contest/ansible/roles/domlogo/templates/domlogo.service.j2

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description="DOMlogo"
+After=network.target
+
+[Service]
+Type=simple
+
+Environment=REQUESTS_CA_BUNDLE=/usr/local/share/ca-certificates/domserver.crt
+Environment=DISPLAY=:0
+WorkingDirectory={{ DJ_DIR }}
+ExecStart=/usr/bin/python3 ./domlogo.py
```

Comment:
```suggestion
ExecStart=domlogo.py
```
I assume domlogo has a shebang and otherwise that's something we should fix. If we ever change domlogo away from python this would still work.

Reply:
It does have shebang: https://github.com/DOMjudge/domjudge-scripts/blob/main/domlogo/domlogo.py#L1